theme/theme.inc

```diff
@@ -264,7 +264,7 @@ function islandora_solr_metadata_query_fields($object, &$solr_fields) {
               ));
             }
           }
-          if (islandora_solr_is_date_field($solr_field) && !empty($solr_fields[$solr_field]['date_format'])) {
+          if (!empty($solr_fields[$solr_field]['date_format']) && islandora_solr_is_date_field($solr_fields[$solr_field])) {
```

Comment:
[`islandora_solr_is_date_field()`](https://github.com/Islandora/islandora_solr_search/blob/7.x/includes/utilities.inc#L633) is expecting the name of the field as its argument, I think this is passing the array of values for the field.
I haven't tried this but wondering if it should be 
```
if (!empty($solr_fields[$solr_field]['date_format']) && islandora_solr_is_date_field($solr_field)) {
```

Reply:
100% right, I'm an idiot.

Reply:
no worries